Sources/ParseSwift/Protocols/ParseObjectMutable.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ParseObjectMutable.swift
+//  ParseSwift
+//
+//  Created by Damian Van de Kauter on 30/10/2021.
+//  Copyright © 2021 Parse Community. All rights reserved.
+//
+
+import Foundation
+
+/**
+ The `ParseObjectMutable` protocol adds an empty version of the respective object.
+ This can be used when you only need to update a subset of the fields of an object
+ as oppose to updating every field of an object.
+ Using an empty object and updating a subset of the fields reduces the amount of data
+ sent between client and server when using `save` and `saveAll`
+ to update objects.
+ 
+ **Example use case:**
+ ````
+ var user = User.current?.mutable
+ user?.customKey = "newValue"
+
+ do {
+     try await user?.save()
+ } catch {
+     // Handle error
+ }
+ ````
+
+ - warning: Using the `ParseObjectMutable` protocol requires the developer to
```

Comment:
```suggestion
 - warning: Using the `ParseObjectMutable` protocol requires the developer to initialize all of the `ParseObject` properties. This can be accomplished by making all properties  optional or setting default values for non-optional properties. This also allows your objects to be used as Parse `Pointer`‘s.
```

Reply:
Note: If you click the `Files Changed` tab to review all of my recommendations you can batch all of my suggestions into one commit and then pull them in Xcode. This is done by clicking `Add suggestion to batch`.